### DISCUSSION TITLE
[BUG] El PBTaskAddEditor por accessRole invita tanto a usuarios inactivos como activos

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#132] [4. Bug Reports] [answered]
Me he dado cuenta que si se usa ese Bot mediante accessRole, traera todos los usuarios activos / inactivos que cuentan con ese rol. Me di cuenta cunado estaba revisando los canales de proyectos y había gente inactiva en proyectos más recientes.

No sé si es un bug o una feature, pero creo que falta agregar una regla que busque todos aquellos que se encuentren activos.

Gracias y atento a sus comentarios 🤪

**Comment by acedward** [comment, marked as answer]
@NicolasPLG por tu descripción, parece un bug. Vamos a investigar.

**Reply by NicolasPLG** [reply]
gracias @acedward quedare atento
